automated install/basic-install.sh

```diff
@@ -1544,7 +1544,7 @@ create_pihole_user() {
             # then create and add her to the pihole group
             local str="Creating user 'pihole'"
             printf "%b  %b %s..." "${OVER}" "${INFO}" "${str}"
-            if useradd -r --no-user-group -g pihole -s /usr/sbin/nologin pihole; then
+            if useradd -r --no-user-group -g pihole -s /sbin/nologin pihole; then
                 printf "%b  %b %s\\n" "${OVER}" "${TICK}" "${str}"
```

Comment:
This change and the corresponding one in line 1562 will cause the essentially the same problem, but on other distros.

On debian and its offspring `nologin` lives at `/usr/sbin/nologin`, and is not available from `/sbin`. I assume likewise for the red hat family, but on openSUSE it's reachable from both locations.

Reply:
There is no symlink on a bare metal unbuntu 22.04. It is there on a fresh install of Debian Trixie, but not on a system that has been upgraded from Buster to Bullseye to Bookworm.